pkg/operator/storageclasscontroller/storageclasscontroller.go

```diff
@@ -145,6 +147,20 @@ func (c *StorageClassController) Sync(ctx context.Context, connections []*vclib.
 	return c.updateConditions(ctx, checkResult, overallClusterStatus)
 }
 
+func (c *StorageClassController) SyncRemove(ctx context.Context) error {
+	scString := string(c.manifest)
+	pairs := []string{
+		"${STORAGE_POLICY_NAME}", "sample",
+	}
+
+	policyReplacer := strings.NewReplacer(pairs...)
+	scString = policyReplacer.Replace(scString)
```

Comment:
In my experiments `resourceapply.DeleteStorageClass()` below works equally fine if we replace `${STORAGE_POLICY_NAME}` or don't. Can we remove L152-157 altogether?

Reply:
fixed.